{
  "name": "useerrorsender",
  "version": "1.0.1",
  "description": "this hook uses for sends errors to a url that gets as an argument. this hook send two kind of errors: - errors you handle them your self. use **sendError** method that returns from hook to send these errors. - errors you don't handle them and maybe fires unexpectedly.",
  "main": "useErrorSender.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/faridEsnaashari/useErrorSender.git"
  },
  "keywords": [
    "react",
    "error",
    "api",
    "javascript"
  ],
  "author": "faridEsnaashari",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/faridEsnaashari/useErrorSender/issues"
  },
  "homepage": "https://github.com/faridEsnaashari/useErrorSender#readme"
}
